config/routes.rb

```diff
@@ -51,8 +51,14 @@
 
     resources :companies, only: [:edit, :update]
 
+    resources :tags, only: [:index]
+    post 'tags/search'
```

Comment:
You can probably write this as below.

```suggestion
    resources :tags, only: [:index] do
      get :search, on: :collection
    end
```

Also, should search be a `get` endpoint?

Reply:
It's a `post` endpoint since I'm passing a JSON body to it.

Reply:
Ah cool.